### DISCUSSION TITLE
Hotel door

### DISCUSSION THREAD

**Discussion by gabac** [alexandercerutti/passkit-generator#95] [Q&A] [answered]
Has this been done for the NFC part of the pass in the wallet?
Very hard to find some documentation around this feature...
https://www.businesswire.com/news/home/20211208005152/en/Hyatt-Rolls-Out-Room-Keys-in-Apple-Wallet/

**Comment by alexandercerutti** [comment, marked as answer]
Hey @gabac, I'm not sure about all this. I think you should ask for an NFC certificate to Apple (there's a specific contact form), through which you have to specify why you need them, the terminals model etc.

All the rest of the details is kinda... top-secret, or behind a legal agreement.

About passkit-generator, it has support for the NFC property, that you can use to geenrate an NFC-enabled pass.

Let me know.
